package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@jupiterone/integration-template",
-  "version": "0.0.0",
-  "description": "A JupiterOne Integration",
+  "name": "@jupiterone/graph-atspoke",
+  "version": "1.0.0",
```

Comment:
Just curious. I have historically used `0.1.0` as the first beta release of graph projects, but I also know many projects are released as `1.0.0`. Is this the convention we should be using? Thoughts @austinkelleher @mknoedel @aiwilliams

Reply:
I've been torn in my opinion on this over time. I've settled on thinking it should be `1.0.0` so that we can release minor versions that add additional ingestion. I believe a `0.1.0` -> `0.2.0` change is considered similar to a `1.0.0` -> `2.0.0` change.